docs/release/release-process.md

```diff
@@ -192,6 +201,77 @@ success, all release artifacts will be officially published.
 to Docker Hub.
 
+### Template Stack Release Process
+
+The Template Stacks we maintain are slightly different from the
+controller-based stacks that we maintain. Their processes are similar
+but a little simpler. This section will walk through how to release the
+Template Stacks themselves, and does not directly apply to core
+Crossplane.
+
+For Template Stacks, we do not use release branches unless we need to
+make a patch release. In the future we may need a more robust branching
+strategy, but for now we are not using branches because it is simpler.
+
+Note that Template Stacks **do not** require any code changes to update
+the version.
+
+### Template Stack Tag And Publish Pipeline
+
+Here is the list of all template stacks:
+
+* `stack-minimal-gcp`
+* `stack-minimal-aws`
+* `stack-minimal-azure`
+* `app-wordpress`
+
+Each one should be released as part of a complete release.
+
+#### Tag the Template Stack
+
+Once a template stack is tested and ready for cutting a semver release,
+we will want to tag the repository with the new release version. In most
+cases, to get the version, take a look at the most recent tag in the
+repo, and increment the minor version. For example, if the most recent
+tag was `v0.2.0`, the new tag should be `v0.3.0`.
+
+Run the template stack's tag job on Jenkins, against the `master`
+branch. Enter in the new tag to use. If the current release candidate is
+not the head of `master`, enter in the commit to tag.
+
+You can find the tag pipeline for the individual stack by going to the
+[crossplane org in
+Jenkins](https://jenkinsci.upbound.io/job/crossplane/), finding the
+folder with the same name as the template stack, and then going to the
+`tag` job group. Then going to the `master` branch job under the group.
+For example, here is [a link to the stack-minimal-gcp tag job for
+master](https://jenkinsci.upbound.io/job/crossplane/job/stack-minimal-gcp/job/tag/job/master/).
+
+> **Note:** The first time you run a pipeline on a new branch, you won't get prompted for the values
+> to input. The build will quickly fail and then you can run (not replay) it a second time to be
+> prompted.  This is a Jenkins bug that is tracked by
+> [#41929](https://issues.jenkins-ci.org/browse/JENKINS-41929) and has been open for almost 3 years,
+> so don't hold your breath.
+
+#### Publish the Template Stack
+
+After the tag pipeline has been run and the repository has been tagged,
+you can run the `publish` job for the template stack. For example,
+here's a [link to the stack-minimal-gcp publish
+job](https://jenkinsci.upbound.io/job/crossplane/job/stack-minimal-gcp/job/publish/job/master/).
+This will kick off the official release build and upon success, all
+release artifacts will be officially published. This should also be run
+from the `master` branch in most cases. Or, if a release branch was
+used, from the release branch. The tag parameter should be used, and the
```

Comment:
help me understand in more detail (or point me to the appropriate doc) how the version is generated for the built and published template stack image.  We tagged the release commit in master in the previous step, and in this publishing step we are providing the version tag again.  How does the publishing step use that input? Does it use it directly for the version of the image it will publish, or does it instead use it to find the right commit to build/publish?

Along those lines, are variables like this `STACK_VERSION` for app-wordpress still needed? or are those used in the automated pipelines? https://github.com/crossplane/app-wordpress/blob/master/stack.env#L4